service/src/main/java/bio/terra/workspace/service/workspace/flight/create/workspace/CreateWorkspaceV2Flight.java

```diff
@@ -52,15 +52,18 @@ public CreateWorkspaceV2Flight(FlightMap inputParameters, Object applicationCont
         new CreateWorkspaceStartStep(workspace, appContext.getWorkspaceDao(), wsmResourceStateRule),
         dbRetryRule);
 
+    if (appContext.getFeatureConfiguration().isTpsEnabled()) {
```

Comment:
It was originally flagged off due to differences between the Broad and Verily deployments of WSM. It can likely be removed given the current state of affairs. I'd like to minimize any potential operational impact so I'm going to leave as-is for now, we can evaluate as a tech debt [item](https://broadworkbench.atlassian.net/browse/WOR-1440) in planning.

Reply:
Sounds like a good plan!  (I think that ticket link is incorrect.)

Reply:
Ah, this one: https://broadworkbench.atlassian.net/browse/WOR-1441